ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -2545,7 +2545,17 @@ tests:
       TAG_VERSION: '@4.13'
     test:
     - chain: openshift-e2e-test-qe
-    workflow: cucushift-installer-rehearse-vsphere-ipi-external-lb
+    workflow: cucushift-installer-rehearse-vsphere-ipi-external-lb-pre
```

Comment:
pls update the job name accordingly to reflect the workflow change. e.g: `vsphere-ipi-external-lb-pre-p2-f7`